src/main/kotlin/no/nav/syfo/personoppgave/PersonOppgaveQueries.kt

```diff
@@ -213,20 +214,27 @@ fun Connection.createBehandletPersonoppgave(
     }
 }
 
-const val querySetUbehandletDialogmotesvarOppgave =
+const val queryUpdatePersonoppgave =
     """
     UPDATE PERSON_OPPGAVE
-    SET behandlet_tidspunkt = null, behandlet_veileder_ident = null, sist_endret = ?
-    WHERE referanse_uuid = ?
+    SET behandlet_tidspunkt = ?, behandlet_veileder_ident = ?, sist_endret = ?, publish = ?, published_at = ?
+    WHERE uuid = ?
     """
 
-fun Connection.updateDialogmotesvarOppgaveSetUbehandlet(
-    // TODO: se på navn
-    dialogmotesvar: Dialogmotesvar,
+fun Connection.updatePersonoppgave(
+    personoppgave: PersonOppgave,
 ) {
-    val behandletOppgaver = prepareStatement(querySetUbehandletDialogmotesvarOppgave).use {
-        it.setTimestamp(1, Timestamp.from(dialogmotesvar.svarReceivedAt.toInstant()))
-        it.setString(2, dialogmotesvar.moteuuid.toString())
+    val behandletTidspunkt =
+        if (personoppgave.behandletTidspunkt == null) null else Timestamp.valueOf(personoppgave.behandletTidspunkt)
+    val publishedAt =
+        if (personoppgave.publishedAt == null) null else Timestamp.from(personoppgave.publishedAt.toInstant())
+    val behandletOppgaver = prepareStatement(queryUpdatePersonoppgave).use {
+        it.setTimestamp(1, behandletTidspunkt)
+        it.setString(2, personoppgave.behandletVeilederIdent)
+        it.setTimestamp(3, Timestamp.valueOf(personoppgave.sistEndret))
+        it.setBoolean(4, personoppgave.publish)
+        it.setTimestamp(5, publishedAt)
```

Comment:
Siden det er timestampz og publishedAt er OffsetDateTime skal det være: it.setObject(3, publishedAt)